src/activity_handlers.cpp

```diff
@@ -4103,7 +4103,7 @@ void activity_handlers::fertilize_plot_do_turn( player_activity *act, player *p
         if( have_fertilizer() ) {
             iexamine::fertilize_plant( p, tile, fertilizer );
             if( !have_fertilizer() ) {
-                add_msg( m_info, _( "You have run out of %s" ), fertilizer );
+                add_msg( m_info, _( "You have run out of %s." ), item( fertilizer ).tname() );
```

Comment:
```suggestion
                add_msg( m_info, _( "You have run out of %s." ), item::nname( fertilizer ) );
```
There is no need to create a full `item` instance just to get the item type name.

Reply:
Thanks, appreciate it!